scanner/enricher/nvd/nvd.go

```diff
@@ -126,8 +149,11 @@ func (*Enricher) Name() string {
 }
 
 // FetchEnrichment implements driver.EnrichmentUpdater.
-func (e *Enricher) FetchEnrichment(ctx context.Context, hint driver.Fingerprint) (io.ReadCloser, driver.Fingerprint, error) {
+func (e *Enricher) FetchEnrichment(ctx context.Context, _ driver.Fingerprint) (io.ReadCloser, driver.Fingerprint, error) {
 	ctx = zlog.ContextWithValues(ctx, "component", "enricher/nvd/Enricher/FetchEnrichment")
+	// Force a new hint, to signal updaters that this is new data.
+	hint := driver.Fingerprint(uuid.NewV4().String())
```

Comment:
why not just return a new one at the end? Also was ignoring it the wrong thing to do previously?

Reply:
> Also was ignoring it the wrong thing to do previously?

I haven't confirmed, but I was mostly sure that the vulnerability loader would ignore any new updates because the fingerprint would always be the same (empty). I did this for safety.